torch/_guards.py

```diff
@@ -482,6 +482,11 @@ def __init__(self, fake_mode):
         # this is only set after aot_autograd
         self.fw_metadata = None
         self.params_flat = None
+        # this is for extended return calling convention from backend
+        # compiler to aot_autograd
+        # Per output, what the compiler specified stride of the output is,
+        # or None if no stride is known
+        self.output_strides: Optional[List[Optional[List[int]]]] = None
```

Comment:
Right now we do layout optimization only if dynamic shape is disabled. But we are working on resolving that (blocked on the split reduction support). So, should we set the type here to consider SymInt?

Reply:
yeah this can have symints, will amend